snapper/AsciiFile.h

```diff
@@ -80,13 +82,14 @@ namespace snapper
 
     protected:
 
-	const string Name_C;
-	const bool remove_empty;
-
 	vector<string> Lines_C;
 
-    };
+    private:
 
+	string Name_C;
+	bool remove_empty;
```

Comment:
I know that it is just copy of already existing, but naming looks quite inconsistent, does it follow any C++ style guide?